ra/ra.go

```diff
@@ -778,24 +778,29 @@ func (ra *RegistrationAuthorityImpl) checkLimits(ctx context.Context, names []st
 }
 
 // UpdateRegistration updates an existing Registration with new values.
-func (ra *RegistrationAuthorityImpl) UpdateRegistration(ctx context.Context, base core.Registration, update core.Registration) (reg core.Registration, err error) {
-	base.MergeUpdate(update)
+func (ra *RegistrationAuthorityImpl) UpdateRegistration(ctx context.Context, base core.Registration, update core.Registration) (core.Registration, error) {
+	if changed := base.MergeUpdate(update); !changed {
```

Comment:
Since we aren't actually using `changed` anywhere this can be simplified to `!base.MergeUpdate(update)`.